### PR TITLE
Added OpenCV

### LINKED ISSUE #3341: Request to add icon for "OpenCV"
**Name:** OpenCV
**Website:** https://opencv.org/
**Alexa rank:** 16,634
  

**Official resources for icon and color:** https://opencv.org/resources/media-kit/

### PR DESCRIPTION
**Pull request by ed-muthiah**
image

**Issue:** Closes #3341
**Alexa rank:** [23,310](https://www.alexa.com/siteinfo/opencv.org)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG used from OpenCV media kit and made monochrome.

### CODE REVIEW

File: icons/opencv.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OpenCV icon</title><path d="M24 17.8c0 3.4-2.6 6.2-5.7 6.2-3.2 0-5.7-2.8-5.7-6.2 0-2.3 1.1-4.2 2.8-5.3l1.7 3.1c-.7.4-1.2 1.3-1.2 2.2 0 1.4 1.1 2.5 2.4 2.5 1.3 0 2.3-1.1 2.3-2.5 0-.9-.5-1.7-1.2-2.2l1.7-3.1c1.8 1.1 2.9 3.1 2.9 5.3zM5.7 20.4c-1.3 0-2.4-1.1-2.4-2.5s1.1-2.5 2.4-2.5c.3 0 .7.1 1 .2.1 0 .2 0 .2-.1l1.6-3c-.8-.5-1.8-.8-2.8-.8-3.2 0-5.7 2.8-5.7 6.2C0 21.3 2.6 24 5.7 24c3.2 0 5.8-2.9 5.7-6.3H8c.2 1.4-.9 2.7-2.3 2.7zm8.6-14.2c0 .9-.5 1.7-1.2 2.2l1.7 3.1c1.7-1.1 2.8-3.1 2.8-5.3C17.6 2.8 15 0 11.9 0S6.2 2.8 6.2 6.2c0 2.3 1.1 4.2 2.8 5.3l1.7-3.1c-.7-.5-1.1-1.3-1.1-2.2 0-1.4 1.1-2.5 2.4-2.5 1.2 0 2.3 1.1 2.3 2.5z"/></svg>
```
Comment (adamrusted) [inline icons/opencv.svg:1]: Hi @ed-muthiah, thanks for the contribution!
Unfortunately, it looks like the icon has been stretched ahead of submission, see your icon (red) against the source icon (black). Any chance you could take another look, keeping the aspect ratio of the original icon?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/opencv.svg (hunk 1 above)
Comment (ed-muthiah) [reply icons/opencv.svg:1]: @adamrusted having some trouble resizing to 0 0 24 24 inkscape. What's the process you use to resize on inkscape?

```
x icons\opencv.svg x icon-size Size of <path> must be exactly 24 in one dimension; the size is currently 24.003 x 22.295                   x icon-centered <path> must be centered at (12, 12); the center is currently (12.006, 12.006)   ```

Comment (adamrusted) [reply icons/opencv.svg:1]: @ed-muthiah no worries! So I generally take the source icon in 'unify' the paths into one objects, and then with the single object selected head to the top of the screen where you'll see this:
image

Change the measurement to px, click the lock (keeps aspect ratio) and enter 24 into whichever is the highest value. You can then change the 'artboard' size using `Ctrl + Shift + D` or going to `File > Document Settings` and set the document size to 24px x 24px. You can then use the align tool (see icon below) to center both horizontally and vertically to the page.
image
image

Comment (ed-muthiah) [reply icons/opencv.svg:1]: Ah! I was using 24 pt rather than 24px. One moment.

Comment (ed-muthiah) [reply icons/opencv.svg]: @adamalston howzat!

Comment (adamrusted) [reply icons/opencv.svg]: Beautiful! 👌🏻
Is there any chance you could get it working with 3 decimals of precision in SVGOMG?

Comment (ed-muthiah) [reply icons/opencv.svg]: Seem to get the same error at 3 precision
 ``` x icon-size Size of <path> must be exactly 24 in one dimension; the size is currently 24.077 x 22.298

  x icon-centered <path> must be centered at (12, 12); the center is currently (11.963, 12.001)```

Comment (adamrusted) [reply icons/opencv.svg]: No worries - I've just tried myself and it seems to distort the circles somewhat, so I guess we're sticking with 4! Thanks for your work on this @ed-muthiah 🎉

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @ed-muthiah on your first contribution! 🚀

**Pull request merged**